src/core/blockchain/zilliqa/tokens/zrc2-client.ts

```diff
@@ -34,22 +34,34 @@ export class Zrc2Client {
     }
 
     public async getTokenInfo(contractAddress) {
-        const smartContractSubState = await this.client.getSmartContractSubState(
-            contractAddress,
-            'implementation'
-        );
+        const smartContractSubState = await this.client
+            .getSmartContractSubState(contractAddress, 'implementation')
+            .catch(e => {
```

Comment:
this is ok to fail, the pattern that we use everywhere is to wrap the code in `try...catch`

Reply:
Yup, moved catch to proper  location.